axelor-sale/src/main/java/com/axelor/apps/sale/service/saleorderline/subline/SubSaleOrderLineComputeServiceImpl.java

```diff
@@ -65,10 +66,7 @@ public void computeSumSubLineList(SaleOrderLine saleOrderLine, SaleOrder saleOrd
               subSaleOrderLineList.stream()
                   .map(SaleOrderLine::getSubTotalCostPrice)
                   .reduce(BigDecimal.ZERO, BigDecimal::add)));
```

Comment:
to avoid duplicated code in `SubSaleOrderLineComputeServiceProductionImpl`,  please define the method `computePrices` here and change the implementation in `SubSaleOrderLineComputeServiceProductionImpl`, instead of overriding `computeSumSubLineList`